scripts/linters/pylint_extensions.py

```diff
@@ -1248,52 +1226,17 @@ class DivisionOperatorChecker(checkers.BaseChecker):
         )
     }
 
-    def process_module(self, node):
-        """Process a module to ensure that the division operator('/') is not
-        used and python_utils.divide() is used instead.
+    def visit_assign(self, node):
```

Comment:
Will it catch `print('%d', 2/5)`?

Reply:
I think you can find a better node to catch this, maybe BinOP?

Reply:
Changed to BinOP